src/components/BaseSelect.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+    <div>
+        <c-dropdown auto-close="outside"
```

Comment:
Do you want this for the base select? I thought clicking inside for a single select should close the dropdown? 

With this set what is the behaviour when you select an item from the drop down?

Reply:
Looking at this in the single select PR looks like this isn't an issue. When we click within the single select it is closing nicely, where that closing is handled by SingleSelect. Is this set to `outside` here to that when you click an arrow in the hierarchy it does not close the dropdown?